core/src/scene/sceneLoader.cpp

```diff
@@ -1011,9 +1031,9 @@ void SceneLoader::loadSource(const std::shared_ptr<Platform>& platform, const st
         // If we have MBTiles, we know the source is tiled.
         tiled = true;
         // Create an MBTiles data source from the file at the url and add it to the source chain.
-        rawSources->setNext(std::make_unique<MBTilesDataSource>(platform, name, url, ""));
+        rawSources->setNext(std::make_unique<MBTilesDataSource>(platform, name, std::move(urls), ""));
```

Comment:
`MBTilesDataSource` should not use URL subdomains, since it makes (at most) one network request. We can note this in the relevant documentation as well.